browser/main/modals/PreferencesModal/index.js

```diff
@@ -94,19 +98,26 @@ class Preferences extends React.Component {
     return node.getBoundingClientRect()
   }
 
+  haveToSaveNotif (tab) {
+    return (
+      <p styleName={`saving--${tab[tab.label].type}`}>{tab[tab.label].message}</p>
```

Comment:
It seems that this function only need two variable.
Could you write like this?

```javascript
haveToSaveNotif (type, message) {
  return (
    //
  )
}
```

It will be easier to maintain.